access/src/test/java/ca/nrc/cadc/accesscontrol/GenerateTestKey.java

```diff
@@ -0,0 +1,46 @@
+package ca.nrc.cadc.accesscontrol;
+
+import java.io.FileOutputStream;
+import java.security.KeyPair;
+import java.security.KeyPairGenerator;
+import java.security.PrivateKey;
+import java.security.PublicKey;
+import java.util.Base64;
+
+/**
+ * Generate test RSA key pair
+ */
+public class GenerateTestKey {
+    
```

Comment:
This whole class is not really necessary. I think you could have added a single line to the test setup (like a static setup method in the abstract test class) with a call to the code in cadc-util that I mentioned. There's no need to replicate all that stuff here. There is no need to implement a `main()` and it's certainly less desireable than having it run automatically at the start of test execution (and put keys in the `build` dir rather)